bigbluebutton-html5/imports/ui/components/error-screen/component.jsx

```diff
@@ -6,19 +6,15 @@ import styles from './styles.scss';
 const intlMessages = defineMessages({
```

Comment:
Our default is not someone else's default. If someone wants to fully translate the strings into French and set that as the default you can't because the defaults are hard coded in. It also means having two (or more if you use a string twice) places to update the string if you want to change it.